Formula/iodine.rb

```diff
@@ -0,0 +1,22 @@
+  desc "Tunnel IPv4 traffic through a DNS server"
+  homepage "https://code.kryo.se/iodine"
+  url "https://github.com/yarrick/iodine/archive/refs/tags/v0.7.0.tar.gz"
+  sha256 "7281e2301804e48029877bab27134f7c0eb04567da4d21a6fcbaa7265cb5849e"
+  license "ISC"
+  head "https://github.com/yarrick/iodine.git", branch: "master"
+  depends_on "cmake" => :build
+
+  uses_from_macos "zlib"
+
+  def install
+    system "make"
```

Comment:
```suggestion
```
Do we need the separate `make` step? It looks like you can just do `make install` directly.

Reply:
Yes, you are right. I've verified on my private build without the separate `make` step too.